pkg/utils/test/scheme/scheme.go

```diff
@@ -25,6 +26,7 @@ var (
 	DefaultAddToSchemes = []func(*runtime.Scheme) error{
 		clientgoscheme.AddToScheme,
 		routev1.AddToScheme,
+		userv1.AddToScheme,
```

Comment:
Why is this needed?

Reply:
it's used to mock the creation of a user:
```
user := &userv1.User{
		ObjectMeta: metav1.ObjectMeta{
			Name: encodedUserName,
			Labels: map[string]string{
				"test.opendatahub.io/metrics-auth": "true",
			},
		},
		Groups: []string{groupName},
	}
```
in `createTestGroup()`